test/prompt/test_prompt_node.py

```diff
@@ -260,15 +260,17 @@ def test_run_invalid_template(prompt_node):
 
 @pytest.mark.integration
 def test_invalid_prompting(prompt_node):
-    with pytest.raises(ValueError, match="Hey there, what is the best city in the worl"):
-        prompt_node.prompt(
-            "Hey there, what is the best city in the world?" "Hey there, what is the best city in the world?"
-        )
-
```

Comment:
This is now possible as we assume that this is a prompt_text now. See added test below.